api/src/main/java/org/openmrs/module/reporting/cohort/definition/evaluator/EncounterCohortDefinitionEvaluator.java

```diff
@@ -67,4 +71,11 @@ public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationCon
     	}
     	return new EvaluatedCohort(c, cohortDefinition, context);
     }
+

```

Comment:
Similar to above, this is also repeated quite a bit.  If there were an abstract class, we could simply move this there and all could utilize it.  If there were an interface, we could have a single utility method that would take in that interface and return the relevant locations.  We could add that into DefinitionUtil like the other utility methods you created.